manager/manager.py

```diff
@@ -249,31 +249,23 @@ def start_job(self, job_id):
                     if error.startswith('Bad Request: '):
                         error = error[len('Bad Request: '):]
                     job.error_message(error)
-
-            if 'Payload' in json_data:
-                payload = json_data['Payload']
-
-                print('Payload:')
-                print(json.dumps(payload))
-
-                for message in payload['log']:
-                    if message:
-                        job.log_message(message)
-                for message in payload['errors']:
-                    if message:
-                        job.error_message(message)
-                for message in payload['warnings']:
-                    if message:
-                        job.warning_message(message)
-
-                success = payload['success']
-
-                if payload['errors']:
-                    job.log_message('{0} function returned with errors.'.format(module.name))
-                elif payload['warnings']:
-                    job.log_message('{0} function returned with warnings.'.format(module.name))
-                elif payload['log']:
-                    job.log_message('{0} function returned.'.format(module.name))
+                elif 'success' in json_data:
```

Comment:
Also have question about the line 
```
     if 'errorMessage' in json_data:
                     error = json_data['errorMessage']	
```
Isn't manager just doing an http request here?  Shouldn't we be looking at response.status_code?  Can't find where 'errorMessage' might be set.  Is this left over from an older implementation?